pcdsdevices/epics_motor.py

```diff
@@ -412,3 +404,66 @@ class PMC100(PCDSMotorBase):
 
     def home(self, *args, **kwargs):
         raise NotImplementedError("PMC100 motors have no homing procedure")
+
+
+class Beckhoff(EpicsMotorInterface):
```

Comment:
Not in love with just `Beckhoff` as the name but it does follow precedent...

Reply:
The part of precedent that is broken here is that we use the `Beckhoff` rails for vacuum too, so there is ambiguity. We have no such issues with `IMS` and `Newport`.

Some suggested renames:
`BeckhoffMotor`
`BeckhoffAxis`

Reply:
What I want to do here is rename all of them to `<brand>Motor` and leave `<brand>` as an alias.

Reply:
Though this may be confusing and it will lead to verbose repetition in the autodoc

Reply:
I like `BeckhoffAxis`.

Also, so-so on the full rename. Seems unnecessarily verbose for most of them.